deadbelly/breathEZ
# Tm/feature/faq

### What does this PR do? 
>Clearly and concisely describe the feature. 

This PR updates the `FAQ` component adding in the return the facts about AQI, Pollutants & Mask Type with coverage.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

No problems occurred.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?

`src/Components/Faq/Faq.js` lines: 5-70;

### How should this be tested?
>What steps need to be taken to confirm functionality? 

Look at the file in this pull request as it needs to be connected to the `App `component before it can render to the site.

### Steps To Take
>Any questions and possible steps moving forward.

Once it has been connected to `App` I will then go back and style this component

### This PR is related to which issues:
Closes #7
